gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless.gradle.kts

```diff
@@ -25,7 +25,16 @@ spotless {
 
     format("misc") {
         // define the files to apply `misc` to
-        target("*.md", ".gitignore")
+        target("docs/*.md", "/*.md", "*.md", "test-clients/*.md", ".gitignore")
+        prettier() // Override the default prettier config to limit column width to 100 characters
```

Comment:
The use of `prettier()` as a formatter here means that every developer machine and CI runner will need to have an appropriate version of NodeJS installed and available on the global `PATH`. 

This is not really ideal since it also impacts all potential community contributors. 

It would be better to split this into a separate spotless configuration and use the [Flexmark](https://github.com/diffplug/spotless/tree/main/plugin-gradle#flexmark) plugin.

Reply:
Replaced prettier with Flexmark - Although it seems that we can't configure flexmark to use anything other than the default options so the assumption is that this should be compliant.

Reply:
Another option would be the FreshMark plugin, if we need additional configurability.